src/components/ArticleListCard.jsx

```diff
@@ -9,14 +13,13 @@ function ArticleListCard({ article }) {
     article_img_url,
     comment_count,
   } = article;
+  const handleClick = () => {
+    navigate(`/article/${article_id}`);
```

Comment:
Usually, we only redirect users when they need to be moved from a page after submitting, for example, or a similar action. I can see why you did this due to the div not having a link, but I think having a hyperlink to click through would be a much more simple solution